winui/Kanban/Overview.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: About WinUI Kanban control | Syncfusion
+description: Learn here all about introduction of Syncfusion Essential Studio WinUI Kanban (SfKanban) control, its features and more.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# WinUI Kanban (SfKanban) Overview
```

Comment:
Overview of WinUI Kanban (SfKanban)

Reply:
Changed to Overview of WinUI Kanban (SfKanban)